cli/src/pcluster/cli/commands/pcluster3_configuration_converter.py

```diff
@@ -0,0 +1,593 @@
+        if iam_lambda_role:
+            role["LambdaFunctionRole"] = _role_name_to_arn(iam_lambda_role)
+        if role:
+            iam["Role"] = role
+        if iam:
+            self.pcluster3_configuration[section] = iam
+
+    def convert_addtional_packages(self, section):
+        additional_packages = {}
+        intel_software = {}
+        self.convert_single_field(
+            self.cluster_section_name, "enable_intel_hpc_platform", intel_software, "IntelHpcPlatform", "getboolean"
+        )
+        if intel_software:
+            additional_packages["IntelSoftware"] = intel_software
+        if additional_packages:
+            self.pcluster3_configuration[section] = additional_packages
+
+    def convert_tags(self, section):
+        tags = self.config_parser.get(self.cluster_section_name, "tags", fallback=None)
+        if tags:
+            try:
+                tags = json.loads(tags)
+            except Exception as e:
+                self.error(f"Tags should be in the format of dictionary, {e}")
+            self.pcluster3_configuration["Tags"] = []
+            for key, value in tags.items():
+                self.pcluster3_configuration[section].append({"Key": key, "Value": value})
+
+    def convert_shared_storage(self, section):
+        shared_storage = []
+        function_section_mappings = {
+            self.convert_ebs_settings: "EbsSettings",
+            self.convert_raid_settings: "EbsSettings",
+            self.convert_efs_settings: "EfsSettings",
+            self.convert_fsx_settings: "FsxLustreSettings",
+        }
+        for function, section_name in function_section_mappings.items():
+            storage = function(section_name)
+            if storage:
+                shared_storage += storage
+        if shared_storage:
+            self.pcluster3_configuration[section] = shared_storage
+
+    def convert_storage_base(self, storage_type, label, additional_items=[]):
+        storage_section = {}
+        settings = {}
+        storage_section["Name"] = label
+        storage_type_mapping = {"ebs": "Ebs", "raid": "Ebs", "fsx": "FsxLustre", "efs": "Efs"}
+        storage_section["StorageType"] = storage_type_mapping[storage_type]
+        section_label = f"{storage_type} {label}"
+        for item in [
+            (section_label, "shared_dir", storage_section, "MountDir"),
+            (section_label, "encrypted", settings, "Encrypted", "getboolean"),
+        ]:
+            self.convert_single_field(*item)
+        for item in additional_items:
+            pc2_label, pc3_label = item[0], item[1]
+            conversion_type = item[2] if len(item) == 3 else None
+            self.convert_single_field(section_label, pc2_label, settings, pc3_label, conversion_type)
+        return storage_section, settings, section_label
+
+    def convert_ebs_settings(self, section):
+        ebs_settings = self.config_parser.get(self.cluster_section_name, "ebs_settings", fallback=None)
```

Comment:
This line is repeated quite a few times through this file -- perhaps it's better to make a helper function so that you can simply call `self.config_get("ebs_settings")` and not repeat the same parameters each time (at least where the parameters are the same)

Reply:
Added a helper function: 
def cluster_config_get(self, param):
        return self.config_parser.get(self.cluster_section_name, param, fallback=None)